tests/issues/issue702.icry

```diff
@@ -0,0 +1 @@
+:sat \(x : {b:Bit, a:Bit}) -> x.a == x.b
```

Comment:
It's probably better if we choose an example with a unique satisfying assignment, so that the test output won't vary depending on which witness the solver decides to give us. So maybe we should change `x.a == x.b` to `x.a && x.b` instead.

Reply:
Good point.